packages/components/src/docs/General/6-Tokens.mdx

```diff
@@ -5,8 +5,12 @@ import theme from 'tailwind-theme';
 
 <Meta title="Docs/General/Tokens" />
 
+## Theme Generator
+
 ## Colors
 
+> Override colors in single stories with our Solid Theme Generator: _[Button / Variant x Size](../?path=/story/components-sd-button--variant-and-size) → Show Addons (A) → Switch to tab "🎨 Theme"_
```

Comment:
I would move this under the theme generator headline

Reply:
I removed the headline completely. Thanks for the headsup. Leftover...